docs/spec/v1alpha1/imagepolicies.md

```diff
@@ -124,5 +124,73 @@ The `LatestImage` field contains the image selected by the policy rule, when it
 There is one condition that may be present: the GitOps toolkit-standard `ReadyCondition`. This will
 be marked as true when the policy rule has selected an image.
 
-[image-automation-controller]: https://github.com/image-automation-controller
-[semver-range]: https://github.com/Masterminds/semver#checking-version-constraints
+## Examples
+
+Select the latest `dev` branch build tagged as `<BRANCH>-<BUILD-ID>` (alphabetical):
+
+```yaml
+kind: ImagePolicy
+spec:
+  filterTags:
+    pattern: '^dev-(?P<id>.*)'
+    extract: '$id'
```

Comment:
Is this meaningfully different to the Minio example?

Reply:
I've seen this pattern in many setups `<branch>-<build-id>` that's why I put it here. I can remove it if you want, it is similar to Minio but easier to grok.

Reply:
OK yeah -- sounds like it's meaningfully different because it's a use seen in the wild, and not obviously the same as using CalVer, even though it technically works the same.